ProcessMaker/Models/ProcessRequest.php

```diff
@@ -632,7 +632,11 @@ private function valueAliasRequester($value, $expression)
         $user = User::where('username', $value)->get()->first();
 
         if ($user) {
-            $requests = ProcessRequest::where('user_id', $expression->operator, $user->id)->get();
+            $requests = ProcessRequest::select('id')
+                ->where('user_id', $expression->operator, $user->id)
+                ->distinct()
```

Comment:
Since `id` is the primary key of `process_requests`, `distinct()` should not be required, the generated query is:
```
"select distinct `id` from `process_requests` where `user_id` = ?"
```